components/sysutils/ngrep/ngrep.p5m

```diff
@@ -13,6 +13,8 @@
 # Copyright 2016 Adam Stevko
 #
 
+<transform file path=usr.*/man/.+ -> default mangler.man.stability uncommitted>
```

Comment:
If it's already present in transforms/defaults, we should drop it, otherwise LGTM. i forgot about this, thanks for fix!